Uno.Gallery/Uno.Gallery.Shared/Controls/SamplePageLayout.cs

```diff
@@ -166,10 +168,22 @@ void OnScrolled(object sender, ScrollViewerViewChangedEventArgs e)
 
 		private void OnShareClicked(Hyperlink sender, HyperlinkClickEventArgs args)
 		{
-#if (__IOS__ || __ANDROID__) && !NET6_0_OR_GREATER
-			var sample = DataContext as Sample;
-			_ = Deeplinking.BranchService.Instance.ShareSample(sample, _design);
-#endif
+			if (DataTransferManager.IsSupported())
+			{
+				var dataTransferManager = DataTransferManagerHelper.GetForCurrentView();
+				dataTransferManager.DataRequested += DataRequested_URI;
+				DataTransferManagerHelper.ShowShareUI();
+			}
+		}
+
+		private void DataRequested_URI(DataTransferManager sender, DataRequestedEventArgs args)
+		{
+			args.Request.Data.Properties.Title = "Uno Gallery - Share-Sample Title";
+			args.Request.Data.Properties.Description = "See this awesome project:";
+
+			args.Request.Data.SetWebLink(new Uri("https://gallery.platform.uno/"));
```

Comment:
This doesn't share yhe sample I believe?

Reply:
when you say the sample... what do you mean?? what this does is shares the URL :/ 
what was the old behavior??

Reply:
The old behavior is that the shared link corresponds to a specific sample and design (fluent, material, cupertino, agnostic).

Opening the link should then open the app on the specific sample.

See my PR for the approach I took for this.